Supporting dotnet 10.0 preview version (#1305)
devcontainers/features: src/dotnet/README.md

| aspNetCoreRuntimeVersions | Enter additional ASP.NET Core runtime versions, separated by commas. Use 'latest' for the latest version, 'lts' for the latest LTS version, 'X.Y' or 'X.Y.Z' for a specific version. | string | - |
| workloads | Enter additional .NET SDK workloads, separated by commas. Use 'dotnet workload search' to learn what workloads are available to install. | string | - |

> The dotnet '10.0' option currently points to the '10.0-preview' version of dotnet. The dotnet '10.0' stable version is expected to be released by end of 2025. The preview version is still under development and is generally not supported for production use.

Reviewer: Since you have changed the version name to `preview` you can delete this

Author: Hello @AlvaroRausell ,

I have removed it now.

> The dotnet '10.0' version provided as option is only the dotnet '10.0-preview' version. The dotnet '10.0' stable version is expected to be released by end of 2025. The preview version is still under development and is generally not supported for production use.

Reviewer 2: Grammatic correction as simplifications as we discussed

```suggestion
The dotnet '10.0' option currently points to the '10.0-preview' version of dotnet. The dotnet '10.0' stable version is expected to be released by end of 2025. The preview version is still under development and is generally not supported for production use.
```

Author: Hello @eljog ,

Thank you for the suggestion. I have corrected the same. Kindly let me know in case of further concern.
